packages/html/src/panelbar/panelbar-item.tsx

```diff
@@ -0,0 +1,101 @@
+import { Icon } from '../icon';
+import { States, classNames, stateClassNames } from '../misc';
+
+export const PANELBARITEM_CLASSNAME = `k-panelbar-item`;
+
+const states = [
+    States.hover,
+    States.focus,
+    States.selected,
+    States.disabled
+];
+
+const options = {};
+
+export type KendoPanelBarItemProps = {
+    header?: boolean;
+    level?: number;
+    expanded?: boolean;
+    icon?: string;
+    iconClass?: string;
+    text?: string;
+};
+
+export type PanelBarItemState = { [K in (typeof states)[number]]?: boolean };
+
+const defaultProps = {
+    level: 0,
+    expanded: false,
+    text: "Item"
+};
+
+export const PanelBarItem = (
+    props: KendoPanelBarItemProps &
+    React.HTMLAttributes<HTMLLIElement> &
+    PanelBarItemState
+) => {
+    const {
+        header,
+        level = defaultProps.level,
+        hover,
+        focus,
+        selected,
+        disabled,
+        expanded,
+        icon,
+        iconClass,
+        text,
+        children,
+        ...other
+    } = props;
+
+    return (
+        <li
+            {...other}
+            className={classNames(
+                props.className,
+                PANELBARITEM_CLASSNAME,
+                {
+                    ["k-panelbar-header"]: header,
+                    [`k-level-${level}`]: level || level === 0,
+                    ["k-expanded"]: expanded
+                }
+            )}
+        >
+            <span
+                className={classNames(
+                    "k-link",
+                    stateClassNames("k-link", {
```

Comment:
"k-link" is duplicated?

Reply:
The `stateClassNames` function accepts 2 parameters, `_componentClassName`, and `props`, that's why `k-link` is passed inside and used for a second time. There is no extra `k-link` class added to this element.